db/migrate/20220209070445_add_qa_vote_count_to_question_answer_comments.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AddQaVoteCountToQuestionAnswerComments < ActiveRecord::Migration[6.1]
+  def up
+    add_column :question_answer_comments, :qa_vote_count, :integer, default: 0
```

Comment:
I don't really like the name `qa_vote_count` here since the model reflects the plugin name. However, we have `Post#qa_vote_count` and our code will have one less conditional if we maintain the column name parity.